extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -202,10 +203,17 @@ export class ImageWithRegionsEditorComponent implements OnInit {
 
   ngOnInit(): void {
     this.alwaysEditable = true;
-    // The initializeEditor function is written separately since it
-    // is also called in resetEditor function.
-    this.initializeEditor();
-    this.imageValueChanged(this.value.imagePath);
+    // The following is check is to prevent cases when value not defined.
+    // This is a dynamically created component and in some cases we get
+    // undefined. This happens when ngOnInit runs before we can assign
+    // this.value the value that is supposed to be passed from object editor
+    // component.
```

Comment:
```suggestion
    // The following check is used to prevent cases when the value is not defined.
    // This is a dynamically created component and in some cases we get
    // undefined. This happens when ngOnInit runs before we can assign
    // this.value the value that is supposed to be passed from object editor
    // component.
```

Reply:
Done